psi4/src/psi4/libscf_solver/rhf.cc

```diff
@@ -87,9 +87,10 @@ void RHF::common_init()
     // Allocate matrix memory
     Fa_        = SharedMatrix(factory_->create_matrix("F"));
     Fb_        = Fa_;
-    Ca_        = SharedMatrix(factory_->create_matrix("C"));
+    Ca_        = SharedMatrix(factory_->create_matrix("MO coefficients (C)"));
```

Comment:
I think these new names must be identical to that of the replaced names in `writer.cc` for this for `fichk` to work?